src/main/java/search/ingester/models/Document.java

```diff
@@ -14,37 +14,49 @@
 
 @NotBlankIfAnotherFieldIsBlank(fieldName="content", dependFieldName = "fileBase64")
 public class Document {
+    
     @NotBlank
     //@Pattern(regexp = "^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$", message = "ID must be a UUID")
     @JsonbProperty("id")
     private String id;
+
     @NotBlank
-    @Pattern(regexp = "datahub|website|mhc", flags = Pattern.Flag.CASE_INSENSITIVE, message = "Site must be datahub|website|mhc")
+    @Pattern(regexp = "datahub|website|mhc|sac", flags = Pattern.Flag.CASE_INSENSITIVE, message = "Site must be datahub|website|mhc|sac")
     @JsonbProperty("site")
     private String site;
+
     @NotBlank
     @JsonbProperty("title")
     private String title;
+
     @NotEmpty
     @JsonbProperty("keywords")
     private List<Keyword> keywords;
 
     @JsonbProperty("content")
     private String content;
 
+    @JsonbProperty("content_truncated")
+    private String contentTruncated;
+
     @JsonbProperty("file_base64")
     private String fileBase64;
+
     @JsonbProperty("file_bytes")
     private Integer fileBytes;
+
     @JsonbProperty("file_extension")
     private String fileExtension;
 
     //@NotBlank
     @JsonbProperty("url")
     private String url;
-    //@NotBlank
-    @JsonbProperty(value = "data_type", nillable = true)
-    private String dataType;
+
+    // todo: what was this?
```

Comment:
This was to contain the data type of the document (i.e. dataset / series / publication / file-type?)

Reply:
OK, deleted comment lines and added support for resource_type (and parent_resource_type).